LiteCore/Storage/SQLiteKeyStore.cc

```diff
@@ -233,6 +233,18 @@ namespace litecore {
         return ss.str();
     }
 
+    string SQLiteKeyStore::untransformCollectionName(std::string_view name) {
```

Comment:
Why introduce a new method, rather than using the above method? I don't think we allow backslash in the collection names.